templates/docs/examples/patterns/newsletter-signup/2-col.html

```diff
@@ -0,0 +1,49 @@
+{% extends "_layouts/examples.html" %}
+{% from '_macros/vf_newsletter-signup.jinja' import vf_newsletter_signup %}
+{% block title %}Newsletter signup{% endblock %}
+{% block standalone_css %}patterns_all{% endblock %}
+{% block content %}
+    {% call(slot) vf_newsletter_signup(layout="2-col", title_text="Newsletter signup", form_id="mktoForm_4960", return_url="https://ubuntu.com/#contact-form-success", input_label='Work email', checkbox_id='canonicalUpdatesOptIn', checkbox_label='I agree to receive information about Canonical\'s products and services.') %}
+        {%- if slot == 'description' -%}
+            <p>Get the latest Ubuntu news and updates in your inbox.</p>
+        {%- endif -%}
+        {%- if slot == 'addendum' -%}
+            <p>
+                By submitting this form, I confirm that I have read and agree to <a href="https://ubuntu.com/legal/terms-and-policies/privacy-policy">Canonical's Privacy Policy</a>.
+            </p>
+        {%- endif -%}
```

Comment:
You now have a default addendum slot, so you don't strictly need to include this slot in your examples anymore.